imsv-docs-astro/src/content/docs/guides/funding-protocols/flexi-algorand-funding-protocol.mdoc

```diff
@@ -0,0 +1,226 @@
+---
+title: Algorand Funding Protocol
+description: Overview of Immersve's Algorand Funding Protocol.
+slug: guides/algorand-funding-protocol
+fundingProtocol:
+  source: https://github.com/immersve/funding-contract-algorand
+---
+
+{% warning %}
+**Note**: This functionality is currently available for Developer Preview only
+and not for general use.
+{% /warning %}
+
+The Immersve Flexi Algorand {% link page="guides/funding-protocols"
+title="funding protocol" /%} allows client applications to fund Immersve cards
+via simple native Algorand {% link
+href="https://developer.algorand.org/docs/get-details/asa" title="Asset" /%}
+transfers.
+
+Depositors can withdraw funds at any time with signed withdrawal approvals.
+Deposits are scoped to a single Immersve partner application. Depositor funds
+can only be settled to Immersve's approved settlement address.
+
+## Smart Contract
+
+The Flexi Algorand protocol has a "Master" contract and many partner "Partner
```

Comment:
The double quotes don't feel right.
Is this a way to introduce a new proper noun?
Perhaps only the first mention should use double quotes?